src/main/java/org/fundacionjala/coding/jimmy/CesarCrypto.java

```diff
@@ -0,0 +1,31 @@
+package org.fundacionjala.coding.jimmy;
+
+
+/**
+ * Class for developing the methods.
+ */
+public class CesarCrypto {
+
+    /**
+     * Encode method for CesarCrypto.
+     *
+     * @param texto is the string to encryption.
+     * @param clave is the number for doing the encryption.
+     * @return the encrypted code.
+     */
+    public String encode(String texto, int clave) {
```

Comment:
It is missing the vigenere cipher
also you should implement the encode and decode.